Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM mcr.microsoft.com/dotnet/core/runtime:3.1 AS base

```

Comment:
Is there a more recent container base? If this is .NET Core 3.1 it's very old

Reply:
Updated to have no version, will use latest. Application being used as "self-contained", should work with latest version.